src/function/signal/zpk2tf.js

```diff
@@ -0,0 +1,85 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'zpk2tf'
+
+const dependencies = [
+  'typed',
+  'add',
+  'multiply',
+  'Complex',
+  'number'
+]
+
+export const createZpk2tf = /* #__PURE__ */ factory(name, dependencies, ({ typed, add, multiply, Complex, number }) => {
+  /**
+     * Compute the transfer function of a zero-pole-gain model.
+     *
+     * Syntax:
+     *      math.zpk2tf(z, p, k)
+     *
+     * Examples:
+     *    math.zpk2tf([1, 2], [-1, -2], 1)    // returns [[1, -3, 2], [1, 3, 2]]
+     *
+     * See also:
+     *   freqz
+     * @param {Array} z
```

Comment:
When running the tests locally I saw a warning the documentation of `zpk2tf`. Can you please:

1. put an empty line above `@param {Array} z`
2. add a description to all @param and the @return (like you did for `freqz` too)